docs/02-app/01-building-your-application/05-styling/01-css-modules.mdx

```diff
@@ -347,7 +347,10 @@ To maintain a predictable order, we recommend the following:
 - Extract shared styles into a separate shared component.
 - If using [Tailwind](/docs/app/building-your-application/styling/tailwind-css), import the stylesheet at the top of the file, preferably in the [Root Layout](/docs/app/building-your-application/routing/layouts-and-templates#root-layout-required).
 
-> **Good to know:** CSS ordering behaves differently in development mode, always ensure to check preview deployments to verify the final CSS order in your production build.
+> **Good to know:**
+>
+> - CSS ordering behaves differently in development mode, always ensure to check preview deployments to verify the final CSS order in your production build.
```

Comment:
Should this be `can behave differently`? I feel like our <picture data-single-emoji=":docs:" title=":docs:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/docs/172e1b9b7d137b8a.png" alt=":docs:" align="absmiddle"></picture> (I think this is mentioned elsewhere) is incorrect here—we should be mentioning to check "preview deployments"—I believe the local `next build` should be fine. I don't think we need to deploy to see the final CSS. It would be concerning if this was a "behaves" differently versus "can."